### DISCUSSION TITLE
Ability to read a `tag` (or `marking`) from the page content and use it the header of the page

### DISCUSSION THREAD

**Discussion by ramkumarkb** [typst/typst#1780] [Ideas]
I am not sure if Typst can already do this - usually for mutli-page table or other log running content spanning mutiple pages, the header of a page has a `tag` or `marking` that reflects the content of that page. Example - an account number for which the transactions span mutiple pages or could be any content the author wishes to put in the header. The main idea is this data is dynamic and read from the content of that respective page.

An example from the `ConTeXt` world is here - https://wiki.contextgarden.net/Command/setmarking

**Comment by andreasKroepelin** [comment]
Something like this?

https://typst.app/docs/reference/meta/query/

**Comment by ramkumarkb** [comment]
@andreasKroepelin - Thank you for the answer! Yes, this looks like it - let me try this out on my usecase of long-running table (that spans mutiple pages) and header should reflect from the table data... 

Thanks again ! 💯
